api/src/main/java/io/strimzi/api/kafka/model/KafkaConnectResources.java

```diff
@@ -153,4 +153,16 @@ public static String buildConfigName(String clusterName) {
     public static String buildName(String clusterName, Long buildVersion) {
         return buildConfigName(clusterName) + "-" + buildVersion;
     }
+
+    /**
+     * Returns the name of the Kafka Connect {@code Build} container.
+     *
+     * @param clusterName  The {@code metadata.name} of the {@code KafkaConnect} resource.
+     * @param isOpenshift Flag for determining, if we are running on Openshift
+     *
+     * @return The name of the corresponding Kafka Connect {@code Build} Container.
+     */
+    public static String buildContainerName(String clusterName, boolean isOpenshift) {
+        return isOpenshift ? "docker-build" : buildPodName(clusterName);
+    }
```

Comment:
I think you should keep this internally in the Utils class somewhere and not add it to the public API.

Reply:
Good point. Done